itemloaders/__init__.py

```diff
@@ -126,37 +136,29 @@ def item(self):
         if self.parent is not None:
             return self.parent.item
         else:
-            return self._local_item
+            if self._item is None:
+                self._item = self.default_item_class(**self._values)
+            return self._item
 
     def nested_xpath(self, xpath, **context):
         """
         Create a nested loader with an xpath selector.
         The supplied selector is applied relative to selector associated
-        with this :class:`ItemLoader`. The nested loader shares the item
-        with the parent :class:`ItemLoader` so calls to :meth:`add_xpath`,
-        :meth:`add_value`, :meth:`replace_value`, etc. will behave as expected.
```

Comment:
I’m not sure if this should go or stay; “shares the item” is not technically accurate any more, but maybe we should simply reword it so that it indicates that `load_item()` returns the same object regardless of whether it is called on the original item loader or in a nested item loader.